core-bundle/src/Resources/contao/classes/Crawl.php

```diff
@@ -99,7 +99,7 @@ public function run()
 
 		$jobId = Input::get('jobId');
 		$queue = $factory->createLazyQueue();
-		$crawLogsDir = sys_get_temp_dir() . '/contao-crawl';
+		$crawLogsDir = sys_get_temp_dir() . '/contao-crawl-' . $jobId;
```

Comment:
Should imo rather be something like `/contao-crawl/md5(%kernel.projectDir%)`.

Reply:
> Note: sys_get_temp_dir() can contain a trailing slash, as visible in this example on Strato.

Mabe something like this, then :slightly_smiling_face: 
```suggestion
		$crawLogsDir = Path::join(sys_get_temp_dir(), 'contao/crawl', md5(System::getContainer()->getParameter('kernel.projectDir'));
```

Reply:
I like the use of Path::join. But its important to use a unique folder in tmp. Otherwise the next user that tries to write into the folder "contao" would also get a "permission denied".

Maybe something like this.

`$crawLogsDir = Path::join(sys_get_temp_dir(), 'contao-crawl-' . $jobId);`

Imo $jobId sounds like the more unique solution.

Reply:
> Otherwise the next user that tries to write into the folder "contao" would also get a "permission denied".

The main problem is, that your hoster is using a wrong tmp folder configuration. Every user needs to have its own tmp folder.

Reply:
I contactet strato to clarify the problem.

Reply:
@bohnmedia see https://github.com/contao/contao/issues/267#issuecomment-453941709

Reply:
> Every user needs to have its own tmp folder.

This is not generally true. Shared folders such as `/tmp` or `/var/lib/php/sessions` often have the [sticky bit](https://en.wikipedia.org/wiki/Sticky_bit) set to protect the files.